### PR TITLE
releases: mark 1.4 as Extended

### PR DESCRIPTION
**Pull request by samuelkarp**

### CODE REVIEW

File: content/releases.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ The current state is available in the following tables:
 | [1.1](https://github.com/containerd/containerd/releases/tag/v1.1.8)  | End of Life | April 23, 2018  | October 23, 2019 |
 | [1.2](https://github.com/containerd/containerd/releases/tag/v1.2.13) | End of Life | October 24, 2018 | October 15, 2020 |
 | [1.3](https://github.com/containerd/containerd/releases/tag/v1.3.10) | End of Life | September 26, 2019  | March 4, 2021 |
-| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.4)  | Active   | August 17, 2020 | November 3, 2021 (Active), February 3, 2022 (Extended) |
+| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.4)  | Extended   | August 17, 2020 | November 3, 2021 (Active), February 3, 2022 (Extended) |
```
Comment (AkihiroSuda) [inline content/releases.md:105]: The file is synced with https://github.com/containerd/containerd/blob/main/RELEASES.md

Could you update https://github.com/containerd/containerd/blob/main/RELEASES.md first?

Comment (samuelkarp) [reply content/releases.md:105]: Thanks for the pointer.  See https://github.com/containerd/containerd/pull/6287.

**Comment by AkihiroSuda** [comment]
Covered by https://github.com/containerd/containerd.io/pull/107

**Comment by AkihiroSuda** [comment]
BTW, maybe we should move RELEASES.md and other docs from `containerd` repo to `containerd.io` repo (or vice versa, either is fine to me)
https://github.com/containerd/containerd.io/issues/80#issuecomment-738561350

**Pull request closed without merging**